### ISSUE TITLE
Ground Truth uses human labelers in three annotations instead of two. from_unlabeled_data_to_deployed_machine_learning_model_ground_truth_demo_image_classification.ipynb

### ISSUE THREAD

**Issue by sermolin** [aws/amazon-sagemaker-examples#867]
The expected behavior (quoting from the notebook) is:

If RUN_FULL_AL_DEMO==True, then the job will proceed in multiple iterations.

Iteration 1: Ground Truth will send out 10 images as 'probes' for human annotation. If these are succesfully annotated, proceed to Iteration 2.
Iteration 2: Send out a batch of MaxConcurrentTaskCount - 10 (in our case, 190) images for human annotation to obtain an active learning training batch.
Iteration 3: Send out another batch of 200 images for human annotation to obtain an active learning validation set.
Iteration 4a: Train a neural net to do auto-labeling. Auto-label as many datapoints as possible.
Iteration 4b: If there is any data leftover, send out at most 200 images for human annotation.
Repeat Iteration 4a and 4b until all data is annotated.

However, when the results are plotted (next cell of the python code), it shows that humans annotated 200 images in iterations 1, 2, 3. 

What is the correct behavior of Ground Truth algorithm?

**Comment by hsl89** [comment]
Dear SageMaker Community,
                 In our attempts to provide this repository with a better level of support                 going forward, we’re closing issues that were opened prior to the v2 release of the SDK.                 This is because we believe that over time many of the issues                 posted were solved with the latest release or other recent changes to the repo.                 This will help us reallocate resources towards issues that are                more likely to still be relevant today.                Some of the issues experienced now can be resolved by referencing the                v2 guide: https://sagemaker.readthedocs.io/en/stable/v2.html                 In this guide, you can find simple solutions to common notebook errors,                 like the renaming of parameters and classes.                If you believe your issue is still ongoing and you have updated error                messaging or other info, please re-open it, and we investigate the issue.
                Best Regards,
                AWS SageMaker Team
